keyboards/sthlmkb/storre/info.json

```diff
@@ -0,0 +1,509 @@
+                {"label": "Delete", "matrix": [3, 0], "x": 15, "y": 2},
+
+                {"label": "Caps Lock", "matrix": [3, 1], "x": 0, "y": 3, "w": 1.75},
+                {"label": "A", "matrix": [3, 2], "x": 1.75, "y": 3},
+                {"label": "S", "matrix": [3, 3], "x": 2.75, "y": 3},
+                {"label": "D", "matrix": [3, 4], "x": 3.75, "y": 3},
+                {"label": "F", "matrix": [3, 5], "x": 4.75, "y": 3},
+                {"label": "G", "matrix": [3, 6], "x": 5.75, "y": 3},
+                {"label": "H", "matrix": [3, 7], "x": 6.75, "y": 3},
+                {"label": "J", "matrix": [3, 8], "x": 7.75, "y": 3},
+                {"label": "K", "matrix": [3, 9], "x": 8.75, "y": 3},
+                {"label": "L", "matrix": [3, 10], "x": 9.75, "y": 3},
+                {"label": ";", "matrix": [3, 11], "x": 10.75, "y": 3},
+                {"label": "\"", "matrix": [3, 13], "x": 11.75, "y": 3},
+                {"label": "Enter", "matrix": [3, 14], "x": 12.75, "y": 3, "w": 2.25},
+                {"label": "PgUp", "matrix": [3, 15], "x": 15, "y": 3},
+
+                {"label": "Shift", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
+                {"label": "/", "matrix": [4, 1], "x": 1.25, "y": 4},
+                {"label": "Z", "matrix": [4, 2], "x": 2.25, "y": 4},
+                {"label": "X", "matrix": [4, 3], "x": 3.25, "y": 4},
+                {"label": "C", "matrix": [4, 4], "x": 4.25, "y": 4},
+                {"label": "V", "matrix": [4, 5], "x": 5.25, "y": 4},
+                {"label": "B", "matrix": [4, 6], "x": 6.25, "y": 4},
+                {"label": "N", "matrix": [4, 7], "x": 7.25, "y": 4},
+                {"label": "M", "matrix": [4, 8], "x": 8.25, "y": 4},
+                {"label": "<", "matrix": [4, 9], "x": 9.25, "y": 4},
+                {"label": ">", "matrix": [4, 10], "x": 10.25, "y": 4},
+                {"label": "?", "matrix": [4, 11], "x": 11.25, "y": 4},
+                {"label": "Shift", "matrix": [4, 12], "x": 12.25, "y": 4, "w": 1.75},
+                {"label": "Up", "matrix": [4, 13], "x": 14, "y": 4},
+                {"label": "PgDn", "matrix": [4, 14], "x": 15, "y": 4},
+
+                {"label": "Ctrl", "matrix": [5, 0], "x": 0, "y": 4, "w": 1.5},
+                {"label": "Alt", "matrix": [5, 1], "x": 1.5, "y": 4},
+                {"label": "Cmd", "matrix": [5, 2], "x": 2.5, "y": 4, "w": 1.5},
+                {"label": "Space", "matrix": [5, 6], "x": 4, "y": 4, "w": 7},
+                {"label": "Cmd", "matrix": [5, 11], "x": 11, "y": 4, "w": 1.5},
+
+                {"label": "Left", "matrix": [5, 12], "x": 13, "y": 5},
+                {"label": "Down", "matrix": [5, 13], "x": 14, "y": 5},
+                {"label": "Right", "matrix": [5, 14], "x": 15, "y": 5}
+            ]
+        }
+    },
+    "encoder": {
+        "rotary": [
+            {"pin_a": "D4", "pin_b": "C6"},
+            {"pin_a": "F4", "pin_b": "F5"}
+        ]
+    }
+    }
```

Comment:
- `debug` feature removed
- `url` link fixed
- encoders moved to top row, as it physical layout

add and changed layouts names to confirm to PR Guidelines:
- LAYOUT_all added
ansi and iso layouts expanded:
- blocker
- blocker_split_bs
- blocker_split_space
- blocker_split_bs_space
- blocker_tsangan
- blocker_tsangan_split_bs
```suggestion
{
    "manufacturer": "STHLM kb",
    "keyboard_name": "storre",
    "url": "https://github.com/mohoyt/sthlmkb-storre",
    "maintainer": "mohoyt",
    "development_board": "promicro",
    "usb": {
         "device_version": "1.0.0",
         "pid": "0x5657",
         "vid": "0x8086"
     },
    "features": {
         "bootmagic": false,
         "command": false,
         "console": false,
         "extrakey": true,
         "mousekey": false,
         "encoder": true,
         "oled": true
    },
    "encoder": {
        "rotary": [
            {"pin_a": "D4", "pin_b": "C6"},
            {"pin_a": "F4", "pin_b": "F5"}
        ]
    },
    "layouts": {
        "LAYOUT_all": {
            "layout": [
                {"label": "ENC1", "matrix": [2, 0], "x": 13.5, "y": 0},
                {"label": "ENC2", "matrix": [2, 1], "x": 15, "y": 0},

                {"label": "Esc", "matrix": [0, 0], "x": 0, "y": 1},
                {"label": "F1", "matrix": [0, 1], "x": 1.5, "y": 1},
                {"label": "F2", "matrix": [0, 2], "x": 2.5, "y": 1},
                {"label": "F3", "matrix": [0, 3], "x": 3.5, "y": 1},
                {"label": "F4", "matrix": [0, 4], "x": 4.5, "y": 1},
                {"label": "F5", "matrix": [0, 5], "x": 6, "y": 1},
                {"label": "F6", "matrix": [0, 6], "x": 7, "y": 1},
                {"label": "F7", "matrix": [0, 7], "x": 8, "y": 1},
                {"label": "F8", "matrix": [0, 8], "x": 9, "y": 1},
                {"label": "F9", "matrix": [0, 9], "x": 10.5, "y": 1},
                {"label": "F10", "matrix": [0, 10], "x": 11.5, "y": 1},
                {"label": "F11", "matrix": [0, 11], "x": 12.5, "y": 1},
                {"label": "F12", "matrix": [0, 12], "x": 13.5, "y": 1},
                {"label": "F13", "matrix": [0, 13], "x": 15, "y": 1},

                {"label": "`", "matrix": [1, 0], "x": 0, "y": 2.25},
                {"label": "1", "matrix": [1, 1], "x": 1, "y": 2.25},
                {"label": "2", "matrix": [1, 2], "x": 2, "y": 2.25},
                {"label": "3", "matrix": [1, 3], "x": 3, "y": 2.25},
                {"label": "4", "matrix": [1, 4], "x": 4, "y": 2.25},
                {"label": "5", "matrix": [1, 5], "x": 5, "y": 2.25},
                {"label": "6", "matrix": [1, 6], "x": 6, "y": 2.25},
                {"label": "7", "matrix": [1, 7], "x": 7, "y": 2.25},
                {"label": "8", "matrix": [1, 8], "x": 8, "y": 2.25},
                {"label": "9", "matrix": [1, 9], "x": 9, "y": 2.25},
                {"label": "0", "matrix": [1, 10], "x": 10, "y": 2.25},
                {"label": "-", "matrix": [1, 11], "x": 11, "y": 2.25},
                {"label": "=", "matrix": [1, 12], "x": 12, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 13], "x": 13, "y": 2.25},
                {"label": "Delete", "matrix": [1, 14], "x": 14, "y": 2.25},
                {"label": "Home", "matrix": [1, 15], "x": 15, "y": 2.25},

                {"label": "Tab", "matrix": [2, 2], "x": 0, "y": 3.25, "w": 1.5},
                {"label": "Q", "matrix": [2, 3], "x": 1.5, "y": 3.25},
                {"label": "W", "matrix": [2, 4], "x": 2.5, "y": 3.25},
                {"label": "E", "matrix": [2, 5], "x": 3.5, "y": 3.25},
                {"label": "R", "matrix": [2, 6], "x": 4.5, "y": 3.25},
                {"label": "T", "matrix": [2, 7], "x": 5.5, "y": 3.25},
                {"label": "Y", "matrix": [2, 8], "x": 6.5, "y": 3.25},
                {"label": "U", "matrix": [2, 9], "x": 7.5, "y": 3.25},
                {"label": "I", "matrix": [2, 10], "x": 8.5, "y": 3.25},
                {"label": "O", "matrix": [2, 11], "x": 9.5, "y": 3.25},
                {"label": "P", "matrix": [2, 12], "x": 10.5, "y": 3.25},
                {"label": "{", "matrix": [2, 13], "x": 11.5, "y": 3.25},
                {"label": "}", "matrix": [2, 14], "x": 12.5, "y": 3.25},
                {"label": "|", "matrix": [2, 15], "x": 13.5, "y": 3.25, "w": 1.5},
                {"label": "Delete", "matrix": [3, 0], "x": 15, "y": 3.25},

                {"label": "Caps Lock", "matrix": [3, 1], "x": 0, "y": 4.25, "w": 1.75},
                {"label": "A", "matrix": [3, 2], "x": 1.75, "y": 4.25},
                {"label": "S", "matrix": [3, 3], "x": 2.75, "y": 4.25},
                {"label": "D", "matrix": [3, 4], "x": 3.75, "y": 4.25},
                {"label": "F", "matrix": [3, 5], "x": 4.75, "y": 4.25},
                {"label": "G", "matrix": [3, 6], "x": 5.75, "y": 4.25},
                {"label": "H", "matrix": [3, 7], "x": 6.75, "y": 4.25},
                {"label": "J", "matrix": [3, 8], "x": 7.75, "y": 4.25},
                {"label": "K", "matrix": [3, 9], "x": 8.75, "y": 4.25},
                {"label": "L", "matrix": [3, 10], "x": 9.75, "y": 4.25},
                {"label": ";", "matrix": [3, 11], "x": 10.75, "y": 4.25},
                {"label": "\"", "matrix": [3, 12], "x": 11.75, "y": 4.25},
                {"label": "NUHS", "matrix": [3, 13], "x": 12.75, "y": 4.25},
                {"label": "Enter", "matrix": [3, 14], "x": 13.75, "y": 4.25, "w": 1.25},
                {"label": "PgUp", "matrix": [3, 15], "x": 15, "y": 4.25},

                {"label": "Shift", "matrix": [4, 0], "x": 0, "y": 5.25, "w": 1.25},
                {"label": "/", "matrix": [4, 1], "x": 1.25, "y": 5.25},
                {"label": "Z", "matrix": [4, 2], "x": 2.25, "y": 5.25},
                {"label": "X", "matrix": [4, 3], "x": 3.25, "y": 5.25},
                {"label": "C", "matrix": [4, 4], "x": 4.25, "y": 5.25},
                {"label": "V", "matrix": [4, 5], "x": 5.25, "y": 5.25},
                {"label": "B", "matrix": [4, 6], "x": 6.25, "y": 5.25},
                {"label": "N", "matrix": [4, 7], "x": 7.25, "y": 5.25},
                {"label": "M", "matrix": [4, 8], "x": 8.25, "y": 5.25},
                {"label": "<", "matrix": [4, 9], "x": 9.25, "y": 5.25},
                {"label": ">", "matrix": [4, 10], "x": 10.25, "y": 5.25},
                {"label": "?", "matrix": [4, 11], "x": 11.25, "y": 5.25},
                {"label": "Shift", "matrix": [4, 12], "x": 12.25, "y": 5.25, "w": 1.75},
                {"label": "Up", "matrix": [4, 13], "x": 14, "y": 5.25},
                {"label": "PgDn", "matrix": [4, 14], "x": 15, "y": 5.25},

                {"label": "Ctrl", "matrix": [5, 0], "x": 0, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 1], "x": 1.25, "y": 6.25, "w": 1.25},
                {"label": "Cmd", "matrix": [5, 2], "x": 2.5, "y": 6.25, "w": 1.25},
                {"label": "Space", "matrix": [5, 4], "x": 3.75, "y": 6.25, "w": 2.25},
                {"label": "Super", "matrix": [5, 6], "x": 6, "y": 6.25, "w": 1.25},
                {"label": "Space", "matrix": [5, 8], "x": 7.25, "y": 6.25, "w": 2.75},
                {"label": "Cmd", "matrix": [5, 10], "x": 10, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 11], "x": 11.25, "y": 6.25, "w": 1.25},

                {"label": "Left", "matrix": [5, 12], "x": 13, "y": 6.25},
                {"label": "Down", "matrix": [5, 13], "x": 14, "y": 6.25},
                {"label": "Right", "matrix": [5, 14], "x": 15, "y": 6.25}
            ]
        },
        "LAYOUT_ansi_blocker": {
            "layout": [
                {"label": "ENC1", "matrix": [2, 0], "x": 13.5, "y": 0},
                {"label": "ENC2", "matrix": [2, 1], "x": 15, "y": 0},

                {"label": "Esc", "matrix": [0, 0], "x": 0, "y": 1},
                {"label": "F1", "matrix": [0, 1], "x": 1.5, "y": 1},
                {"label": "F2", "matrix": [0, 2], "x": 2.5, "y": 1},
                {"label": "F3", "matrix": [0, 3], "x": 3.5, "y": 1},
                {"label": "F4", "matrix": [0, 4], "x": 4.5, "y": 1},
                {"label": "F5", "matrix": [0, 5], "x": 6, "y": 1},
                {"label": "F6", "matrix": [0, 6], "x": 7, "y": 1},
                {"label": "F7", "matrix": [0, 7], "x": 8, "y": 1},
                {"label": "F8", "matrix": [0, 8], "x": 9, "y": 1},
                {"label": "F9", "matrix": [0, 9], "x": 10.5, "y": 1},
                {"label": "F10", "matrix": [0, 10], "x": 11.5, "y": 1},
                {"label": "F11", "matrix": [0, 11], "x": 12.5, "y": 1},
                {"label": "F12", "matrix": [0, 12], "x": 13.5, "y": 1},
                {"label": "F13", "matrix": [0, 13], "x": 15, "y": 1},

                {"label": "`", "matrix": [1, 0], "x": 0, "y": 2.25},
                {"label": "1", "matrix": [1, 1], "x": 1, "y": 2.25},
                {"label": "2", "matrix": [1, 2], "x": 2, "y": 2.25},
                {"label": "3", "matrix": [1, 3], "x": 3, "y": 2.25},
                {"label": "4", "matrix": [1, 4], "x": 4, "y": 2.25},
                {"label": "5", "matrix": [1, 5], "x": 5, "y": 2.25},
                {"label": "6", "matrix": [1, 6], "x": 6, "y": 2.25},
                {"label": "7", "matrix": [1, 7], "x": 7, "y": 2.25},
                {"label": "8", "matrix": [1, 8], "x": 8, "y": 2.25},
                {"label": "9", "matrix": [1, 9], "x": 9, "y": 2.25},
                {"label": "0", "matrix": [1, 10], "x": 10, "y": 2.25},
                {"label": "-", "matrix": [1, 11], "x": 11, "y": 2.25},
                {"label": "=", "matrix": [1, 12], "x": 12, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 14], "x": 13, "y": 2.25, "w": 2},
                {"label": "Home", "matrix": [1, 15], "x": 15, "y": 2.25},      

                {"label": "Tab", "matrix": [2, 2], "x": 0, "y": 3.25, "w": 1.5},
                {"label": "Q", "matrix": [2, 3], "x": 1.5, "y": 3.25},
                {"label": "W", "matrix": [2, 4], "x": 2.5, "y": 3.25},
                {"label": "E", "matrix": [2, 5], "x": 3.5, "y": 3.25},
                {"label": "R", "matrix": [2, 6], "x": 4.5, "y": 3.25},
                {"label": "T", "matrix": [2, 7], "x": 5.5, "y": 3.25},
                {"label": "Y", "matrix": [2, 8], "x": 6.5, "y": 3.25},
                {"label": "U", "matrix": [2, 9], "x": 7.5, "y": 3.25},
                {"label": "I", "matrix": [2, 10], "x": 8.5, "y": 3.25},
                {"label": "O", "matrix": [2, 11], "x": 9.5, "y": 3.25},
                {"label": "P", "matrix": [2, 12], "x": 10.5, "y": 3.25},
                {"label": "{", "matrix": [2, 13], "x": 11.5, "y": 3.25},
                {"label": "}", "matrix": [2, 14], "x": 12.5, "y": 3.25},
                {"label": "|", "matrix": [2, 15], "x": 13.5, "y": 3.25, "w": 1.5},
                {"label": "Delete", "matrix": [3, 0], "x": 15, "y": 3.25},

                {"label": "Caps Lock", "matrix": [3, 1], "x": 0, "y": 4.25, "w": 1.75},
                {"label": "A", "matrix": [3, 2], "x": 1.75, "y": 4.25},
                {"label": "S", "matrix": [3, 3], "x": 2.75, "y": 4.25},
                {"label": "D", "matrix": [3, 4], "x": 3.75, "y": 4.25},
                {"label": "F", "matrix": [3, 5], "x": 4.75, "y": 4.25},
                {"label": "G", "matrix": [3, 6], "x": 5.75, "y": 4.25},
                {"label": "H", "matrix": [3, 7], "x": 6.75, "y": 4.25},
                {"label": "J", "matrix": [3, 8], "x": 7.75, "y": 4.25},
                {"label": "K", "matrix": [3, 9], "x": 8.75, "y": 4.25},
                {"label": "L", "matrix": [3, 10], "x": 9.75, "y": 4.25},
                {"label": ";", "matrix": [3, 11], "x": 10.75, "y": 4.25},
                {"label": "\"", "matrix": [3, 13], "x": 11.75, "y": 4.25},
                {"label": "Enter", "matrix": [3, 14], "x": 12.75, "y": 4.25, "w": 2.25},
                {"label": "PgUp", "matrix": [3, 15], "x": 15, "y": 4.25},

                {"label": "Shift", "matrix": [4, 0], "x": 0, "y": 5.25, "w": 2.25},
                {"label": "Z", "matrix": [4, 2], "x": 2.25, "y": 5.25},
                {"label": "X", "matrix": [4, 3], "x": 3.25, "y": 5.25},
                {"label": "C", "matrix": [4, 4], "x": 4.25, "y": 5.25},
                {"label": "V", "matrix": [4, 5], "x": 5.25, "y": 5.25},
                {"label": "B", "matrix": [4, 6], "x": 6.25, "y": 5.25},
                {"label": "N", "matrix": [4, 7], "x": 7.25, "y": 5.25},
                {"label": "M", "matrix": [4, 8], "x": 8.25, "y": 5.25},
                {"label": "<", "matrix": [4, 9], "x": 9.25, "y": 5.25},
                {"label": ">", "matrix": [4, 10], "x": 10.25, "y": 5.25},
                {"label": "?", "matrix": [4, 11], "x": 11.25, "y": 5.25},
                {"label": "Shift", "matrix": [4, 12], "x": 12.25, "y": 5.25, "w": 1.75},
                {"label": "Up", "matrix": [4, 13], "x": 14, "y": 5.25},
                {"label": "PgDn", "matrix": [4, 14], "x": 15, "y": 5.25},

                {"label": "Ctrl", "matrix": [5, 0], "x": 0, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 1], "x": 1.25, "y": 6.25, "w": 1.25},
                {"label": "Cmd", "matrix": [5, 2], "x": 2.5, "y": 6.25, "w": 1.25},
                {"label": "Space", "matrix": [5, 6], "x": 3.75, "y": 6.25, "w": 6.25},
                {"label": "Cmd", "matrix": [5, 10], "x": 10, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 11], "x": 11.25, "y": 6.25, "w": 1.25},

                {"label": "Left", "matrix": [5, 12], "x": 13, "y": 6.25},
                {"label": "Down", "matrix": [5, 13], "x": 14, "y": 6.25},
                {"label": "Right", "matrix": [5, 14], "x": 15, "y": 6.25}
            ]
        },
        "LAYOUT_ansi_blocker_split_bs": {
            "layout": [
                {"label": "ENC1", "matrix": [2, 0], "x": 13.5, "y": 0},
                {"label": "ENC2", "matrix": [2, 1], "x": 15, "y": 0},

                {"label": "Esc", "matrix": [0, 0], "x": 0, "y": 1},
                {"label": "F1", "matrix": [0, 1], "x": 1.5, "y": 1},
                {"label": "F2", "matrix": [0, 2], "x": 2.5, "y": 1},
                {"label": "F3", "matrix": [0, 3], "x": 3.5, "y": 1},
                {"label": "F4", "matrix": [0, 4], "x": 4.5, "y": 1},
                {"label": "F5", "matrix": [0, 5], "x": 6, "y": 1},
                {"label": "F6", "matrix": [0, 6], "x": 7, "y": 1},
                {"label": "F7", "matrix": [0, 7], "x": 8, "y": 1},
                {"label": "F8", "matrix": [0, 8], "x": 9, "y": 1},
                {"label": "F9", "matrix": [0, 9], "x": 10.5, "y": 1},
                {"label": "F10", "matrix": [0, 10], "x": 11.5, "y": 1},
                {"label": "F11", "matrix": [0, 11], "x": 12.5, "y": 1},
                {"label": "F12", "matrix": [0, 12], "x": 13.5, "y": 1},
                {"label": "F13", "matrix": [0, 13], "x": 15, "y": 1},

                {"label": "`", "matrix": [1, 0], "x": 0, "y": 2.25},
                {"label": "1", "matrix": [1, 1], "x": 1, "y": 2.25},
                {"label": "2", "matrix": [1, 2], "x": 2, "y": 2.25},
                {"label": "3", "matrix": [1, 3], "x": 3, "y": 2.25},
                {"label": "4", "matrix": [1, 4], "x": 4, "y": 2.25},
                {"label": "5", "matrix": [1, 5], "x": 5, "y": 2.25},
                {"label": "6", "matrix": [1, 6], "x": 6, "y": 2.25},
                {"label": "7", "matrix": [1, 7], "x": 7, "y": 2.25},
                {"label": "8", "matrix": [1, 8], "x": 8, "y": 2.25},
                {"label": "9", "matrix": [1, 9], "x": 9, "y": 2.25},
                {"label": "0", "matrix": [1, 10], "x": 10, "y": 2.25},
                {"label": "-", "matrix": [1, 11], "x": 11, "y": 2.25},
                {"label": "=", "matrix": [1, 12], "x": 12, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 13], "x": 13, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 14], "x": 14, "y": 2.25},
                {"label": "Home", "matrix": [1, 15], "x": 15, "y": 2.25},      

                {"label": "Tab", "matrix": [2, 2], "x": 0, "y": 3.25, "w": 1.5},
                {"label": "Q", "matrix": [2, 3], "x": 1.5, "y": 3.25},
                {"label": "W", "matrix": [2, 4], "x": 2.5, "y": 3.25},
                {"label": "E", "matrix": [2, 5], "x": 3.5, "y": 3.25},
                {"label": "R", "matrix": [2, 6], "x": 4.5, "y": 3.25},
                {"label": "T", "matrix": [2, 7], "x": 5.5, "y": 3.25},
                {"label": "Y", "matrix": [2, 8], "x": 6.5, "y": 3.25},
                {"label": "U", "matrix": [2, 9], "x": 7.5, "y": 3.25},
                {"label": "I", "matrix": [2, 10], "x": 8.5, "y": 3.25},
                {"label": "O", "matrix": [2, 11], "x": 9.5, "y": 3.25},
                {"label": "P", "matrix": [2, 12], "x": 10.5, "y": 3.25},
                {"label": "{", "matrix": [2, 13], "x": 11.5, "y": 3.25},
                {"label": "}", "matrix": [2, 14], "x": 12.5, "y": 3.25},
                {"label": "|", "matrix": [2, 15], "x": 13.5, "y": 3.25, "w": 1.5},
                {"label": "Delete", "matrix": [3, 0], "x": 15, "y": 3.25},

                {"label": "Caps Lock", "matrix": [3, 1], "x": 0, "y": 4.25, "w": 1.75},
                {"label": "A", "matrix": [3, 2], "x": 1.75, "y": 4.25},
                {"label": "S", "matrix": [3, 3], "x": 2.75, "y": 4.25},
                {"label": "D", "matrix": [3, 4], "x": 3.75, "y": 4.25},
                {"label": "F", "matrix": [3, 5], "x": 4.75, "y": 4.25},
                {"label": "G", "matrix": [3, 6], "x": 5.75, "y": 4.25},
                {"label": "H", "matrix": [3, 7], "x": 6.75, "y": 4.25},
                {"label": "J", "matrix": [3, 8], "x": 7.75, "y": 4.25},
                {"label": "K", "matrix": [3, 9], "x": 8.75, "y": 4.25},
                {"label": "L", "matrix": [3, 10], "x": 9.75, "y": 4.25},
                {"label": ";", "matrix": [3, 11], "x": 10.75, "y": 4.25},
                {"label": "\"", "matrix": [3, 13], "x": 11.75, "y": 4.25},
                {"label": "Enter", "matrix": [3, 14], "x": 12.75, "y": 4.25, "w": 2.25},
                {"label": "PgUp", "matrix": [3, 15], "x": 15, "y": 4.25},

                {"label": "Shift", "matrix": [4, 0], "x": 0, "y": 5.25, "w": 2.25},
                {"label": "Z", "matrix": [4, 2], "x": 2.25, "y": 5.25},
                {"label": "X", "matrix": [4, 3], "x": 3.25, "y": 5.25},
                {"label": "C", "matrix": [4, 4], "x": 4.25, "y": 5.25},
                {"label": "V", "matrix": [4, 5], "x": 5.25, "y": 5.25},
                {"label": "B", "matrix": [4, 6], "x": 6.25, "y": 5.25},
                {"label": "N", "matrix": [4, 7], "x": 7.25, "y": 5.25},
                {"label": "M", "matrix": [4, 8], "x": 8.25, "y": 5.25},
                {"label": "<", "matrix": [4, 9], "x": 9.25, "y": 5.25},
                {"label": ">", "matrix": [4, 10], "x": 10.25, "y": 5.25},
                {"label": "?", "matrix": [4, 11], "x": 11.25, "y": 5.25},
                {"label": "Shift", "matrix": [4, 12], "x": 12.25, "y": 5.25, "w": 1.75},
                {"label": "Up", "matrix": [4, 13], "x": 14, "y": 5.25},
                {"label": "PgDn", "matrix": [4, 14], "x": 15, "y": 5.25},

                {"label": "Ctrl", "matrix": [5, 0], "x": 0, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 1], "x": 1.25, "y": 6.25, "w": 1.25},
                {"label": "Cmd", "matrix": [5, 2], "x": 2.5, "y": 6.25, "w": 1.25},
                {"label": "Space", "matrix": [5, 6], "x": 3.75, "y": 6.25, "w": 6.25},
                {"label": "Cmd", "matrix": [5, 10], "x": 10, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 11], "x": 11.25, "y": 6.25, "w": 1.25},

                {"label": "Left", "matrix": [5, 12], "x": 13, "y": 6.25},
                {"label": "Down", "matrix": [5, 13], "x": 14, "y": 6.25},
                {"label": "Right", "matrix": [5, 14], "x": 15, "y": 6.25}
            ]
        },
        "LAYOUT_ansi_blocker_split_space": {
            "layout": [
                {"label": "ENC1", "matrix": [2, 0], "x": 13.5, "y": 0},
                {"label": "ENC2", "matrix": [2, 1], "x": 15, "y": 0},

                {"label": "Esc", "matrix": [0, 0], "x": 0, "y": 1},
                {"label": "F1", "matrix": [0, 1], "x": 1.5, "y": 1},
                {"label": "F2", "matrix": [0, 2], "x": 2.5, "y": 1},
                {"label": "F3", "matrix": [0, 3], "x": 3.5, "y": 1},
                {"label": "F4", "matrix": [0, 4], "x": 4.5, "y": 1},
                {"label": "F5", "matrix": [0, 5], "x": 6, "y": 1},
                {"label": "F6", "matrix": [0, 6], "x": 7, "y": 1},
                {"label": "F7", "matrix": [0, 7], "x": 8, "y": 1},
                {"label": "F8", "matrix": [0, 8], "x": 9, "y": 1},
                {"label": "F9", "matrix": [0, 9], "x": 10.5, "y": 1},
                {"label": "F10", "matrix": [0, 10], "x": 11.5, "y": 1},
                {"label": "F11", "matrix": [0, 11], "x": 12.5, "y": 1},
                {"label": "F12", "matrix": [0, 12], "x": 13.5, "y": 1},
                {"label": "F13", "matrix": [0, 13], "x": 15, "y": 1},

                {"label": "`", "matrix": [1, 0], "x": 0, "y": 2.25},
                {"label": "1", "matrix": [1, 1], "x": 1, "y": 2.25},
                {"label": "2", "matrix": [1, 2], "x": 2, "y": 2.25},
                {"label": "3", "matrix": [1, 3], "x": 3, "y": 2.25},
                {"label": "4", "matrix": [1, 4], "x": 4, "y": 2.25},
                {"label": "5", "matrix": [1, 5], "x": 5, "y": 2.25},
                {"label": "6", "matrix": [1, 6], "x": 6, "y": 2.25},
                {"label": "7", "matrix": [1, 7], "x": 7, "y": 2.25},
                {"label": "8", "matrix": [1, 8], "x": 8, "y": 2.25},
                {"label": "9", "matrix": [1, 9], "x": 9, "y": 2.25},
                {"label": "0", "matrix": [1, 10], "x": 10, "y": 2.25},
                {"label": "-", "matrix": [1, 11], "x": 11, "y": 2.25},
                {"label": "=", "matrix": [1, 12], "x": 12, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 14], "x": 13, "y": 2.25, "w": 2},
                {"label": "Home", "matrix": [1, 15], "x": 15, "y": 2.25},      

                {"label": "Tab", "matrix": [2, 2], "x": 0, "y": 3.25, "w": 1.5},
                {"label": "Q", "matrix": [2, 3], "x": 1.5, "y": 3.25},
                {"label": "W", "matrix": [2, 4], "x": 2.5, "y": 3.25},
                {"label": "E", "matrix": [2, 5], "x": 3.5, "y": 3.25},
                {"label": "R", "matrix": [2, 6], "x": 4.5, "y": 3.25},
                {"label": "T", "matrix": [2, 7], "x": 5.5, "y": 3.25},
                {"label": "Y", "matrix": [2, 8], "x": 6.5, "y": 3.25},
                {"label": "U", "matrix": [2, 9], "x": 7.5, "y": 3.25},
                {"label": "I", "matrix": [2, 10], "x": 8.5, "y": 3.25},
                {"label": "O", "matrix": [2, 11], "x": 9.5, "y": 3.25},
                {"label": "P", "matrix": [2, 12], "x": 10.5, "y": 3.25},
                {"label": "{", "matrix": [2, 13], "x": 11.5, "y": 3.25},
                {"label": "}", "matrix": [2, 14], "x": 12.5, "y": 3.25},
                {"label": "|", "matrix": [2, 15], "x": 13.5, "y": 3.25, "w": 1.5},
                {"label": "Delete", "matrix": [3, 0], "x": 15, "y": 3.25},

                {"label": "Caps Lock", "matrix": [3, 1], "x": 0, "y": 4.25, "w": 1.75},
                {"label": "A", "matrix": [3, 2], "x": 1.75, "y": 4.25},
                {"label": "S", "matrix": [3, 3], "x": 2.75, "y": 4.25},
                {"label": "D", "matrix": [3, 4], "x": 3.75, "y": 4.25},
                {"label": "F", "matrix": [3, 5], "x": 4.75, "y": 4.25},
                {"label": "G", "matrix": [3, 6], "x": 5.75, "y": 4.25},
                {"label": "H", "matrix": [3, 7], "x": 6.75, "y": 4.25},
                {"label": "J", "matrix": [3, 8], "x": 7.75, "y": 4.25},
                {"label": "K", "matrix": [3, 9], "x": 8.75, "y": 4.25},
                {"label": "L", "matrix": [3, 10], "x": 9.75, "y": 4.25},
                {"label": ";", "matrix": [3, 11], "x": 10.75, "y": 4.25},
                {"label": "\"", "matrix": [3, 13], "x": 11.75, "y": 4.25},
                {"label": "Enter", "matrix": [3, 14], "x": 12.75, "y": 4.25, "w": 2.25},
                {"label": "PgUp", "matrix": [3, 15], "x": 15, "y": 4.25},

                {"label": "Shift", "matrix": [4, 0], "x": 0, "y": 5.25, "w": 2.25},
                {"label": "Z", "matrix": [4, 2], "x": 2.25, "y": 5.25},
                {"label": "X", "matrix": [4, 3], "x": 3.25, "y": 5.25},
                {"label": "C", "matrix": [4, 4], "x": 4.25, "y": 5.25},
                {"label": "V", "matrix": [4, 5], "x": 5.25, "y": 5.25},
                {"label": "B", "matrix": [4, 6], "x": 6.25, "y": 5.25},
                {"label": "N", "matrix": [4, 7], "x": 7.25, "y": 5.25},
                {"label": "M", "matrix": [4, 8], "x": 8.25, "y": 5.25},
                {"label": "<", "matrix": [4, 9], "x": 9.25, "y": 5.25},
                {"label": ">", "matrix": [4, 10], "x": 10.25, "y": 5.25},
                {"label": "?", "matrix": [4, 11], "x": 11.25, "y": 5.25},
                {"label": "Shift", "matrix": [4, 12], "x": 12.25, "y": 5.25, "w": 1.75},
                {"label": "Up", "matrix": [4, 13], "x": 14, "y": 5.25},
                {"label": "PgDn", "matrix": [4, 14], "x": 15, "y": 5.25},

                {"label": "Ctrl", "matrix": [5, 0], "x": 0, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 1], "x": 1.25, "y": 6.25, "w": 1.25},
                {"label": "Cmd", "matrix": [5, 2], "x": 2.5, "y": 6.25, "w": 1.25},
                {"label": "Space", "matrix": [5, 4], "x": 3.75, "y": 6.25, "w": 2.25},
                {"label": "Super", "matrix": [5, 6], "x": 6, "y": 6.25, "w": 1.25},
                {"label": "Space", "matrix": [5, 8], "x": 7.25, "y": 6.25, "w": 2.75},
                {"label": "Cmd", "matrix": [5, 10], "x": 10, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 11], "x": 11.25, "y": 6.25, "w": 1.25},

                {"label": "Left", "matrix": [5, 12], "x": 13, "y": 6.25},
                {"label": "Down", "matrix": [5, 13], "x": 14, "y": 6.25},
                {"label": "Right", "matrix": [5, 14], "x": 15, "y": 6.25}
            ]
        },
        "LAYOUT_ansi_blocker_split_bs_space": {
            "layout": [
                {"label": "ENC1", "matrix": [2, 0], "x": 13.5, "y": 0},
                {"label": "ENC2", "matrix": [2, 1], "x": 15, "y": 0},

                {"label": "Esc", "matrix": [0, 0], "x": 0, "y": 1},
                {"label": "F1", "matrix": [0, 1], "x": 1.5, "y": 1},
                {"label": "F2", "matrix": [0, 2], "x": 2.5, "y": 1},
                {"label": "F3", "matrix": [0, 3], "x": 3.5, "y": 1},
                {"label": "F4", "matrix": [0, 4], "x": 4.5, "y": 1},
                {"label": "F5", "matrix": [0, 5], "x": 6, "y": 1},
                {"label": "F6", "matrix": [0, 6], "x": 7, "y": 1},
                {"label": "F7", "matrix": [0, 7], "x": 8, "y": 1},
                {"label": "F8", "matrix": [0, 8], "x": 9, "y": 1},
                {"label": "F9", "matrix": [0, 9], "x": 10.5, "y": 1},
                {"label": "F10", "matrix": [0, 10], "x": 11.5, "y": 1},
                {"label": "F11", "matrix": [0, 11], "x": 12.5, "y": 1},
                {"label": "F12", "matrix": [0, 12], "x": 13.5, "y": 1},
                {"label": "F13", "matrix": [0, 13], "x": 15, "y": 1},

                {"label": "`", "matrix": [1, 0], "x": 0, "y": 2.25},
                {"label": "1", "matrix": [1, 1], "x": 1, "y": 2.25},
                {"label": "2", "matrix": [1, 2], "x": 2, "y": 2.25},
                {"label": "3", "matrix": [1, 3], "x": 3, "y": 2.25},
                {"label": "4", "matrix": [1, 4], "x": 4, "y": 2.25},
                {"label": "5", "matrix": [1, 5], "x": 5, "y": 2.25},
                {"label": "6", "matrix": [1, 6], "x": 6, "y": 2.25},
                {"label": "7", "matrix": [1, 7], "x": 7, "y": 2.25},
                {"label": "8", "matrix": [1, 8], "x": 8, "y": 2.25},
                {"label": "9", "matrix": [1, 9], "x": 9, "y": 2.25},
                {"label": "0", "matrix": [1, 10], "x": 10, "y": 2.25},
                {"label": "-", "matrix": [1, 11], "x": 11, "y": 2.25},
                {"label": "=", "matrix": [1, 12], "x": 12, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 13], "x": 13, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 14], "x": 14, "y": 2.25},
                {"label": "Home", "matrix": [1, 15], "x": 15, "y": 2.25},      

                {"label": "Tab", "matrix": [2, 2], "x": 0, "y": 3.25, "w": 1.5},
                {"label": "Q", "matrix": [2, 3], "x": 1.5, "y": 3.25},
                {"label": "W", "matrix": [2, 4], "x": 2.5, "y": 3.25},
                {"label": "E", "matrix": [2, 5], "x": 3.5, "y": 3.25},
                {"label": "R", "matrix": [2, 6], "x": 4.5, "y": 3.25},
                {"label": "T", "matrix": [2, 7], "x": 5.5, "y": 3.25},
                {"label": "Y", "matrix": [2, 8], "x": 6.5, "y": 3.25},
                {"label": "U", "matrix": [2, 9], "x": 7.5, "y": 3.25},
                {"label": "I", "matrix": [2, 10], "x": 8.5, "y": 3.25},
                {"label": "O", "matrix": [2, 11], "x": 9.5, "y": 3.25},
                {"label": "P", "matrix": [2, 12], "x": 10.5, "y": 3.25},
                {"label": "{", "matrix": [2, 13], "x": 11.5, "y": 3.25},
                {"label": "}", "matrix": [2, 14], "x": 12.5, "y": 3.25},
                {"label": "|", "matrix": [2, 15], "x": 13.5, "y": 3.25, "w": 1.5},
                {"label": "Delete", "matrix": [3, 0], "x": 15, "y": 3.25},

                {"label": "Caps Lock", "matrix": [3, 1], "x": 0, "y": 4.25, "w": 1.75},
                {"label": "A", "matrix": [3, 2], "x": 1.75, "y": 4.25},
                {"label": "S", "matrix": [3, 3], "x": 2.75, "y": 4.25},
                {"label": "D", "matrix": [3, 4], "x": 3.75, "y": 4.25},
                {"label": "F", "matrix": [3, 5], "x": 4.75, "y": 4.25},
                {"label": "G", "matrix": [3, 6], "x": 5.75, "y": 4.25},
                {"label": "H", "matrix": [3, 7], "x": 6.75, "y": 4.25},
                {"label": "J", "matrix": [3, 8], "x": 7.75, "y": 4.25},
                {"label": "K", "matrix": [3, 9], "x": 8.75, "y": 4.25},
                {"label": "L", "matrix": [3, 10], "x": 9.75, "y": 4.25},
                {"label": ";", "matrix": [3, 11], "x": 10.75, "y": 4.25},
                {"label": "\"", "matrix": [3, 13], "x": 11.75, "y": 4.25},
                {"label": "Enter", "matrix": [3, 14], "x": 12.75, "y": 4.25, "w": 2.25},
                {"label": "PgUp", "matrix": [3, 15], "x": 15, "y": 4.25},

                {"label": "Shift", "matrix": [4, 0], "x": 0, "y": 5.25, "w": 2.25},
                {"label": "Z", "matrix": [4, 2], "x": 2.25, "y": 5.25},
                {"label": "X", "matrix": [4, 3], "x": 3.25, "y": 5.25},
                {"label": "C", "matrix": [4, 4], "x": 4.25, "y": 5.25},
                {"label": "V", "matrix": [4, 5], "x": 5.25, "y": 5.25},
                {"label": "B", "matrix": [4, 6], "x": 6.25, "y": 5.25},
                {"label": "N", "matrix": [4, 7], "x": 7.25, "y": 5.25},
                {"label": "M", "matrix": [4, 8], "x": 8.25, "y": 5.25},
                {"label": "<", "matrix": [4, 9], "x": 9.25, "y": 5.25},
                {"label": ">", "matrix": [4, 10], "x": 10.25, "y": 5.25},
                {"label": "?", "matrix": [4, 11], "x": 11.25, "y": 5.25},
                {"label": "Shift", "matrix": [4, 12], "x": 12.25, "y": 5.25, "w": 1.75},
                {"label": "Up", "matrix": [4, 13], "x": 14, "y": 5.25},
                {"label": "PgDn", "matrix": [4, 14], "x": 15, "y": 5.25},

                {"label": "Ctrl", "matrix": [5, 0], "x": 0, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 1], "x": 1.25, "y": 6.25, "w": 1.25},
                {"label": "Cmd", "matrix": [5, 2], "x": 2.5, "y": 6.25, "w": 1.25},
                {"label": "Space", "matrix": [5, 4], "x": 3.75, "y": 6.25, "w": 2.25},
                {"label": "Super", "matrix": [5, 6], "x": 6, "y": 6.25, "w": 1.25},
                {"label": "Space", "matrix": [5, 8], "x": 7.25, "y": 6.25, "w": 2.75},
                {"label": "Cmd", "matrix": [5, 10], "x": 10, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 11], "x": 11.25, "y": 6.25, "w": 1.25},

                {"label": "Left", "matrix": [5, 12], "x": 13, "y": 6.25},
                {"label": "Down", "matrix": [5, 13], "x": 14, "y": 6.25},
                {"label": "Right", "matrix": [5, 14], "x": 15, "y": 6.25}
            ]
        },
        "LAYOUT_ansi_blocker_tsangan": {
            "layout": [
                {"label": "ENC1", "matrix": [2, 0], "x": 13.5, "y": 0},
                {"label": "ENC2", "matrix": [2, 1], "x": 15, "y": 0},

                {"label": "Esc", "matrix": [0, 0], "x": 0, "y": 1},
                {"label": "F1", "matrix": [0, 1], "x": 1.5, "y": 1},
                {"label": "F2", "matrix": [0, 2], "x": 2.5, "y": 1},
                {"label": "F3", "matrix": [0, 3], "x": 3.5, "y": 1},
                {"label": "F4", "matrix": [0, 4], "x": 4.5, "y": 1},
                {"label": "F5", "matrix": [0, 5], "x": 6, "y": 1},
                {"label": "F6", "matrix": [0, 6], "x": 7, "y": 1},
                {"label": "F7", "matrix": [0, 7], "x": 8, "y": 1},
                {"label": "F8", "matrix": [0, 8], "x": 9, "y": 1},
                {"label": "F9", "matrix": [0, 9], "x": 10.5, "y": 1},
                {"label": "F10", "matrix": [0, 10], "x": 11.5, "y": 1},
                {"label": "F11", "matrix": [0, 11], "x": 12.5, "y": 1},
                {"label": "F12", "matrix": [0, 12], "x": 13.5, "y": 1},
                {"label": "F13", "matrix": [0, 13], "x": 15, "y": 1},

                {"label": "`", "matrix": [1, 0], "x": 0, "y": 2.25},
                {"label": "1", "matrix": [1, 1], "x": 1, "y": 2.25},
                {"label": "2", "matrix": [1, 2], "x": 2, "y": 2.25},
                {"label": "3", "matrix": [1, 3], "x": 3, "y": 2.25},
                {"label": "4", "matrix": [1, 4], "x": 4, "y": 2.25},
                {"label": "5", "matrix": [1, 5], "x": 5, "y": 2.25},
                {"label": "6", "matrix": [1, 6], "x": 6, "y": 2.25},
                {"label": "7", "matrix": [1, 7], "x": 7, "y": 2.25},
                {"label": "8", "matrix": [1, 8], "x": 8, "y": 2.25},
                {"label": "9", "matrix": [1, 9], "x": 9, "y": 2.25},
                {"label": "0", "matrix": [1, 10], "x": 10, "y": 2.25},
                {"label": "-", "matrix": [1, 11], "x": 11, "y": 2.25},
                {"label": "=", "matrix": [1, 12], "x": 12, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 14], "x": 13, "y": 2.25, "w": 2},
                {"label": "Home", "matrix": [1, 15], "x": 15, "y": 2.25},      

                {"label": "Tab", "matrix": [2, 2], "x": 0, "y": 3.25, "w": 1.5},
                {"label": "Q", "matrix": [2, 3], "x": 1.5, "y": 3.25},
                {"label": "W", "matrix": [2, 4], "x": 2.5, "y": 3.25},
                {"label": "E", "matrix": [2, 5], "x": 3.5, "y": 3.25},
                {"label": "R", "matrix": [2, 6], "x": 4.5, "y": 3.25},
                {"label": "T", "matrix": [2, 7], "x": 5.5, "y": 3.25},
                {"label": "Y", "matrix": [2, 8], "x": 6.5, "y": 3.25},
                {"label": "U", "matrix": [2, 9], "x": 7.5, "y": 3.25},
                {"label": "I", "matrix": [2, 10], "x": 8.5, "y": 3.25},
                {"label": "O", "matrix": [2, 11], "x": 9.5, "y": 3.25},
                {"label": "P", "matrix": [2, 12], "x": 10.5, "y": 3.25},
                {"label": "{", "matrix": [2, 13], "x": 11.5, "y": 3.25},
                {"label": "}", "matrix": [2, 14], "x": 12.5, "y": 3.25},
                {"label": "|", "matrix": [2, 15], "x": 13.5, "y": 3.25, "w": 1.5},
                {"label": "Delete", "matrix": [3, 0], "x": 15, "y": 3.25},

                {"label": "Caps Lock", "matrix": [3, 1], "x": 0, "y": 4.25, "w": 1.75},
                {"label": "A", "matrix": [3, 2], "x": 1.75, "y": 4.25},
                {"label": "S", "matrix": [3, 3], "x": 2.75, "y": 4.25},
                {"label": "D", "matrix": [3, 4], "x": 3.75, "y": 4.25},
                {"label": "F", "matrix": [3, 5], "x": 4.75, "y": 4.25},
                {"label": "G", "matrix": [3, 6], "x": 5.75, "y": 4.25},
                {"label": "H", "matrix": [3, 7], "x": 6.75, "y": 4.25},
                {"label": "J", "matrix": [3, 8], "x": 7.75, "y": 4.25},
                {"label": "K", "matrix": [3, 9], "x": 8.75, "y": 4.25},
                {"label": "L", "matrix": [3, 10], "x": 9.75, "y": 4.25},
                {"label": ";", "matrix": [3, 11], "x": 10.75, "y": 4.25},
                {"label": "\"", "matrix": [3, 13], "x": 11.75, "y": 4.25},
                {"label": "Enter", "matrix": [3, 14], "x": 12.75, "y": 4.25, "w": 2.25},
                {"label": "PgUp", "matrix": [3, 15], "x": 15, "y": 4.25},

                {"label": "Shift", "matrix": [4, 0], "x": 0, "y": 5.25, "w": 2.25},
                {"label": "Z", "matrix": [4, 2], "x": 2.25, "y": 5.25},
                {"label": "X", "matrix": [4, 3], "x": 3.25, "y": 5.25},
                {"label": "C", "matrix": [4, 4], "x": 4.25, "y": 5.25},
                {"label": "V", "matrix": [4, 5], "x": 5.25, "y": 5.25},
                {"label": "B", "matrix": [4, 6], "x": 6.25, "y": 5.25},
                {"label": "N", "matrix": [4, 7], "x": 7.25, "y": 5.25},
                {"label": "M", "matrix": [4, 8], "x": 8.25, "y": 5.25},
                {"label": "<", "matrix": [4, 9], "x": 9.25, "y": 5.25},
                {"label": ">", "matrix": [4, 10], "x": 10.25, "y": 5.25},
                {"label": "?", "matrix": [4, 11], "x": 11.25, "y": 5.25},
                {"label": "Shift", "matrix": [4, 12], "x": 12.25, "y": 5.25, "w": 1.75},
                {"label": "Up", "matrix": [4, 13], "x": 14, "y": 5.25},
                {"label": "PgDn", "matrix": [4, 14], "x": 15, "y": 5.25},

                {"label": "Ctrl", "matrix": [5, 0], "x": 0, "y": 6.25, "w": 1.5},
                {"label": "Alt", "matrix": [5, 1], "x": 1.5, "y": 6.25},
                {"label": "Cmd", "matrix": [5, 2], "x": 2.5, "y": 6.25, "w": 1.5},
                {"label": "Space", "matrix": [5, 6], "x": 4, "y": 6.25, "w": 7},
                {"label": "Cmd", "matrix": [5, 11], "x": 11, "y": 6.25, "w": 1.5},

                {"label": "Left", "matrix": [5, 12], "x": 13, "y": 6.25},
                {"label": "Down", "matrix": [5, 13], "x": 14, "y": 6.25},
                {"label": "Right", "matrix": [5, 14], "x": 15, "y": 6.25}
            ]
        },
        "LAYOUT_ansi_blocker_tsangan_split_bs": {
            "layout": [
                {"label": "ENC1", "matrix": [2, 0], "x": 13.5, "y": 0},
                {"label": "ENC2", "matrix": [2, 1], "x": 15, "y": 0},

                {"label": "Esc", "matrix": [0, 0], "x": 0, "y": 1},
                {"label": "F1", "matrix": [0, 1], "x": 1.5, "y": 1},
                {"label": "F2", "matrix": [0, 2], "x": 2.5, "y": 1},
                {"label": "F3", "matrix": [0, 3], "x": 3.5, "y": 1},
                {"label": "F4", "matrix": [0, 4], "x": 4.5, "y": 1},
                {"label": "F5", "matrix": [0, 5], "x": 6, "y": 1},
                {"label": "F6", "matrix": [0, 6], "x": 7, "y": 1},
                {"label": "F7", "matrix": [0, 7], "x": 8, "y": 1},
                {"label": "F8", "matrix": [0, 8], "x": 9, "y": 1},
                {"label": "F9", "matrix": [0, 9], "x": 10.5, "y": 1},
                {"label": "F10", "matrix": [0, 10], "x": 11.5, "y": 1},
                {"label": "F11", "matrix": [0, 11], "x": 12.5, "y": 1},
                {"label": "F12", "matrix": [0, 12], "x": 13.5, "y": 1},
                {"label": "F13", "matrix": [0, 13], "x": 15, "y": 1},

                {"label": "`", "matrix": [1, 0], "x": 0, "y": 2.25},
                {"label": "1", "matrix": [1, 1], "x": 1, "y": 2.25},
                {"label": "2", "matrix": [1, 2], "x": 2, "y": 2.25},
                {"label": "3", "matrix": [1, 3], "x": 3, "y": 2.25},
                {"label": "4", "matrix": [1, 4], "x": 4, "y": 2.25},
                {"label": "5", "matrix": [1, 5], "x": 5, "y": 2.25},
                {"label": "6", "matrix": [1, 6], "x": 6, "y": 2.25},
                {"label": "7", "matrix": [1, 7], "x": 7, "y": 2.25},
                {"label": "8", "matrix": [1, 8], "x": 8, "y": 2.25},
                {"label": "9", "matrix": [1, 9], "x": 9, "y": 2.25},
                {"label": "0", "matrix": [1, 10], "x": 10, "y": 2.25},
                {"label": "-", "matrix": [1, 11], "x": 11, "y": 2.25},
                {"label": "=", "matrix": [1, 12], "x": 12, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 13], "x": 13, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 14], "x": 14, "y": 2.25},
                {"label": "Home", "matrix": [1, 15], "x": 15, "y": 2.25},      

                {"label": "Tab", "matrix": [2, 2], "x": 0, "y": 3.25, "w": 1.5},
                {"label": "Q", "matrix": [2, 3], "x": 1.5, "y": 3.25},
                {"label": "W", "matrix": [2, 4], "x": 2.5, "y": 3.25},
                {"label": "E", "matrix": [2, 5], "x": 3.5, "y": 3.25},
                {"label": "R", "matrix": [2, 6], "x": 4.5, "y": 3.25},
                {"label": "T", "matrix": [2, 7], "x": 5.5, "y": 3.25},
                {"label": "Y", "matrix": [2, 8], "x": 6.5, "y": 3.25},
                {"label": "U", "matrix": [2, 9], "x": 7.5, "y": 3.25},
                {"label": "I", "matrix": [2, 10], "x": 8.5, "y": 3.25},
                {"label": "O", "matrix": [2, 11], "x": 9.5, "y": 3.25},
                {"label": "P", "matrix": [2, 12], "x": 10.5, "y": 3.25},
                {"label": "{", "matrix": [2, 13], "x": 11.5, "y": 3.25},
                {"label": "}", "matrix": [2, 14], "x": 12.5, "y": 3.25},
                {"label": "|", "matrix": [2, 15], "x": 13.5, "y": 3.25, "w": 1.5},
                {"label": "Delete", "matrix": [3, 0], "x": 15, "y": 3.25},

                {"label": "Caps Lock", "matrix": [3, 1], "x": 0, "y": 4.25, "w": 1.75},
                {"label": "A", "matrix": [3, 2], "x": 1.75, "y": 4.25},
                {"label": "S", "matrix": [3, 3], "x": 2.75, "y": 4.25},
                {"label": "D", "matrix": [3, 4], "x": 3.75, "y": 4.25},
                {"label": "F", "matrix": [3, 5], "x": 4.75, "y": 4.25},
                {"label": "G", "matrix": [3, 6], "x": 5.75, "y": 4.25},
                {"label": "H", "matrix": [3, 7], "x": 6.75, "y": 4.25},
                {"label": "J", "matrix": [3, 8], "x": 7.75, "y": 4.25},
                {"label": "K", "matrix": [3, 9], "x": 8.75, "y": 4.25},
                {"label": "L", "matrix": [3, 10], "x": 9.75, "y": 4.25},
                {"label": ";", "matrix": [3, 11], "x": 10.75, "y": 4.25},
                {"label": "\"", "matrix": [3, 13], "x": 11.75, "y": 4.25},
                {"label": "Enter", "matrix": [3, 14], "x": 12.75, "y": 4.25, "w": 2.25},
                {"label": "PgUp", "matrix": [3, 15], "x": 15, "y": 4.25},

                {"label": "Shift", "matrix": [4, 0], "x": 0, "y": 5.25, "w": 2.25},
                {"label": "Z", "matrix": [4, 2], "x": 2.25, "y": 5.25},
                {"label": "X", "matrix": [4, 3], "x": 3.25, "y": 5.25},
                {"label": "C", "matrix": [4, 4], "x": 4.25, "y": 5.25},
                {"label": "V", "matrix": [4, 5], "x": 5.25, "y": 5.25},
                {"label": "B", "matrix": [4, 6], "x": 6.25, "y": 5.25},
                {"label": "N", "matrix": [4, 7], "x": 7.25, "y": 5.25},
                {"label": "M", "matrix": [4, 8], "x": 8.25, "y": 5.25},
                {"label": "<", "matrix": [4, 9], "x": 9.25, "y": 5.25},
                {"label": ">", "matrix": [4, 10], "x": 10.25, "y": 5.25},
                {"label": "?", "matrix": [4, 11], "x": 11.25, "y": 5.25},
                {"label": "Shift", "matrix": [4, 12], "x": 12.25, "y": 5.25, "w": 1.75},
                {"label": "Up", "matrix": [4, 13], "x": 14, "y": 5.25},
                {"label": "PgDn", "matrix": [4, 14], "x": 15, "y": 5.25},

                {"label": "Ctrl", "matrix": [5, 0], "x": 0, "y": 6.25, "w": 1.5},
                {"label": "Alt", "matrix": [5, 1], "x": 1.5, "y": 6.25},
                {"label": "Cmd", "matrix": [5, 2], "x": 2.5, "y": 6.25, "w": 1.5},
                {"label": "Space", "matrix": [5, 6], "x": 4, "y": 6.25, "w": 7},
                {"label": "Cmd", "matrix": [5, 11], "x": 11, "y": 6.25, "w": 1.5},

                {"label": "Left", "matrix": [5, 12], "x": 13, "y": 6.25},
                {"label": "Down", "matrix": [5, 13], "x": 14, "y": 6.25},
                {"label": "Right", "matrix": [5, 14], "x": 15, "y": 6.25}
            ]
        },
        "LAYOUT_iso_blocker": {
            "layout": [
                {"label": "ENC1", "matrix": [2, 0], "x": 13.5, "y": 0},
                {"label": "ENC2", "matrix": [2, 1], "x": 15, "y": 0},

                {"label": "Esc", "matrix": [0, 0], "x": 0, "y": 1},
                {"label": "F1", "matrix": [0, 1], "x": 1.5, "y": 1},
                {"label": "F2", "matrix": [0, 2], "x": 2.5, "y": 1},
                {"label": "F3", "matrix": [0, 3], "x": 3.5, "y": 1},
                {"label": "F4", "matrix": [0, 4], "x": 4.5, "y": 1},
                {"label": "F5", "matrix": [0, 5], "x": 6, "y": 1},
                {"label": "F6", "matrix": [0, 6], "x": 7, "y": 1},
                {"label": "F7", "matrix": [0, 7], "x": 8, "y": 1},
                {"label": "F8", "matrix": [0, 8], "x": 9, "y": 1},
                {"label": "F9", "matrix": [0, 9], "x": 10.5, "y": 1},
                {"label": "F10", "matrix": [0, 10], "x": 11.5, "y": 1},
                {"label": "F11", "matrix": [0, 11], "x": 12.5, "y": 1},
                {"label": "F12", "matrix": [0, 12], "x": 13.5, "y": 1},
                {"label": "F13", "matrix": [0, 13], "x": 15, "y": 1},

                {"label": "`", "matrix": [1, 0], "x": 0, "y": 2.25},
                {"label": "1", "matrix": [1, 1], "x": 1, "y": 2.25},
                {"label": "2", "matrix": [1, 2], "x": 2, "y": 2.25},
                {"label": "3", "matrix": [1, 3], "x": 3, "y": 2.25},
                {"label": "4", "matrix": [1, 4], "x": 4, "y": 2.25},
                {"label": "5", "matrix": [1, 5], "x": 5, "y": 2.25},
                {"label": "6", "matrix": [1, 6], "x": 6, "y": 2.25},
                {"label": "7", "matrix": [1, 7], "x": 7, "y": 2.25},
                {"label": "8", "matrix": [1, 8], "x": 8, "y": 2.25},
                {"label": "9", "matrix": [1, 9], "x": 9, "y": 2.25},
                {"label": "0", "matrix": [1, 10], "x": 10, "y": 2.25},
                {"label": "-", "matrix": [1, 11], "x": 11, "y": 2.25},
                {"label": "=", "matrix": [1, 12], "x": 12, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 14], "x": 13, "y": 2.25, "w": 2},
                {"label": "Home", "matrix": [1, 15], "x": 15, "y": 2.25},

                {"label": "Tab", "matrix": [2, 2], "x": 0, "y": 3.25, "w": 1.5},
                {"label": "Q", "matrix": [2, 3], "x": 1.5, "y": 3.25},
                {"label": "W", "matrix": [2, 4], "x": 2.5, "y": 3.25},
                {"label": "E", "matrix": [2, 5], "x": 3.5, "y": 3.25},
                {"label": "R", "matrix": [2, 6], "x": 4.5, "y": 3.25},
                {"label": "T", "matrix": [2, 7], "x": 5.5, "y": 3.25},
                {"label": "Y", "matrix": [2, 8], "x": 6.5, "y": 3.25},
                {"label": "U", "matrix": [2, 9], "x": 7.5, "y": 3.25},
                {"label": "I", "matrix": [2, 10], "x": 8.5, "y": 3.25},
                {"label": "O", "matrix": [2, 11], "x": 9.5, "y": 3.25},
                {"label": "P", "matrix": [2, 12], "x": 10.5, "y": 3.25},
                {"label": "{", "matrix": [2, 13], "x": 11.5, "y": 3.25},
                {"label": "}", "matrix": [2, 14], "x": 12.5, "y": 3.25},
                {"label": "Delete", "matrix": [3, 0], "x": 15, "y": 3.25},

                {"label": "Caps Lock", "matrix": [3, 1], "x": 0, "y": 4.25, "w": 1.75},
                {"label": "A", "matrix": [3, 2], "x": 1.75, "y": 4.25},
                {"label": "S", "matrix": [3, 3], "x": 2.75, "y": 4.25},
                {"label": "D", "matrix": [3, 4], "x": 3.75, "y": 4.25},
                {"label": "F", "matrix": [3, 5], "x": 4.75, "y": 4.25},
                {"label": "G", "matrix": [3, 6], "x": 5.75, "y": 4.25},
                {"label": "H", "matrix": [3, 7], "x": 6.75, "y": 4.25},
                {"label": "J", "matrix": [3, 8], "x": 7.75, "y": 4.25},
                {"label": "K", "matrix": [3, 9], "x": 8.75, "y": 4.25},
                {"label": "L", "matrix": [3, 10], "x": 9.75, "y": 4.25},
                {"label": ";", "matrix": [3, 11], "x": 10.75, "y": 4.25},
                {"label": "\"", "matrix": [3, 12], "x": 11.75, "y": 4.25},
                {"label": "NUHS", "matrix": [3, 13], "x": 12.75, "y": 4.25},
                {"label": "Enter", "matrix": [3, 14], "x": 13.75, "y": 3.25, "w": 1.25, "h": 2},
                {"label": "PgUp", "matrix": [3, 15], "x": 15, "y": 4.25},

                {"label": "Shift", "matrix": [4, 0], "x": 0, "y": 5.25, "w": 1.25},
                {"label": "|", "matrix": [4, 1], "x": 1.25, "y": 5.25},
                {"label": "Z", "matrix": [4, 2], "x": 2.25, "y": 5.25},
                {"label": "X", "matrix": [4, 3], "x": 3.25, "y": 5.25},
                {"label": "C", "matrix": [4, 4], "x": 4.25, "y": 5.25},
                {"label": "V", "matrix": [4, 5], "x": 5.25, "y": 5.25},
                {"label": "B", "matrix": [4, 6], "x": 6.25, "y": 5.25},
                {"label": "N", "matrix": [4, 7], "x": 7.25, "y": 5.25},
                {"label": "M", "matrix": [4, 8], "x": 8.25, "y": 5.25},
                {"label": "<", "matrix": [4, 9], "x": 9.25, "y": 5.25},
                {"label": ">", "matrix": [4, 10], "x": 10.25, "y": 5.25},
                {"label": "?", "matrix": [4, 11], "x": 11.25, "y": 5.25},
                {"label": "Shift", "matrix": [4, 12], "x": 12.25, "y": 5.25, "w": 1.75},
                {"label": "Up", "matrix": [4, 13], "x": 14, "y": 5.25},
                {"label": "PgDn", "matrix": [4, 14], "x": 15, "y": 5.25},

                {"label": "Ctrl", "matrix": [5, 0], "x": 0, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 1], "x": 1.25, "y": 6.25, "w": 1.25},
                {"label": "Cmd", "matrix": [5, 2], "x": 2.5, "y": 6.25, "w": 1.25},
                {"label": "Space", "matrix": [5, 6], "x": 3.75, "y": 6.25, "w": 6.25},
                {"label": "Cmd", "matrix": [5, 10], "x": 10, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 11], "x": 11.25, "y": 6.25, "w": 1.25},

                {"label": "Left", "matrix": [5, 12], "x": 13, "y": 6.25},
                {"label": "Down", "matrix": [5, 13], "x": 14, "y": 6.25},
                {"label": "Right", "matrix": [5, 14], "x": 15, "y": 6.25}
            ]
        },
        "LAYOUT_iso_blocker_split_bs": {
            "layout": [
                {"label": "ENC1", "matrix": [2, 0], "x": 13.5, "y": 0},
                {"label": "ENC2", "matrix": [2, 1], "x": 15, "y": 0},

                {"label": "Esc", "matrix": [0, 0], "x": 0, "y": 1},
                {"label": "F1", "matrix": [0, 1], "x": 1.5, "y": 1},
                {"label": "F2", "matrix": [0, 2], "x": 2.5, "y": 1},
                {"label": "F3", "matrix": [0, 3], "x": 3.5, "y": 1},
                {"label": "F4", "matrix": [0, 4], "x": 4.5, "y": 1},
                {"label": "F5", "matrix": [0, 5], "x": 6, "y": 1},
                {"label": "F6", "matrix": [0, 6], "x": 7, "y": 1},
                {"label": "F7", "matrix": [0, 7], "x": 8, "y": 1},
                {"label": "F8", "matrix": [0, 8], "x": 9, "y": 1},
                {"label": "F9", "matrix": [0, 9], "x": 10.5, "y": 1},
                {"label": "F10", "matrix": [0, 10], "x": 11.5, "y": 1},
                {"label": "F11", "matrix": [0, 11], "x": 12.5, "y": 1},
                {"label": "F12", "matrix": [0, 12], "x": 13.5, "y": 1},
                {"label": "F13", "matrix": [0, 13], "x": 15, "y": 1},

                {"label": "`", "matrix": [1, 0], "x": 0, "y": 2.25},
                {"label": "1", "matrix": [1, 1], "x": 1, "y": 2.25},
                {"label": "2", "matrix": [1, 2], "x": 2, "y": 2.25},
                {"label": "3", "matrix": [1, 3], "x": 3, "y": 2.25},
                {"label": "4", "matrix": [1, 4], "x": 4, "y": 2.25},
                {"label": "5", "matrix": [1, 5], "x": 5, "y": 2.25},
                {"label": "6", "matrix": [1, 6], "x": 6, "y": 2.25},
                {"label": "7", "matrix": [1, 7], "x": 7, "y": 2.25},
                {"label": "8", "matrix": [1, 8], "x": 8, "y": 2.25},
                {"label": "9", "matrix": [1, 9], "x": 9, "y": 2.25},
                {"label": "0", "matrix": [1, 10], "x": 10, "y": 2.25},
                {"label": "-", "matrix": [1, 11], "x": 11, "y": 2.25},
                {"label": "=", "matrix": [1, 12], "x": 12, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 13], "x": 13, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 14], "x": 14, "y": 2.25},
                {"label": "Home", "matrix": [1, 15], "x": 15, "y": 2.25},      

                {"label": "Tab", "matrix": [2, 2], "x": 0, "y": 3.25, "w": 1.5},
                {"label": "Q", "matrix": [2, 3], "x": 1.5, "y": 3.25},
                {"label": "W", "matrix": [2, 4], "x": 2.5, "y": 3.25},
                {"label": "E", "matrix": [2, 5], "x": 3.5, "y": 3.25},
                {"label": "R", "matrix": [2, 6], "x": 4.5, "y": 3.25},
                {"label": "T", "matrix": [2, 7], "x": 5.5, "y": 3.25},
                {"label": "Y", "matrix": [2, 8], "x": 6.5, "y": 3.25},
                {"label": "U", "matrix": [2, 9], "x": 7.5, "y": 3.25},
                {"label": "I", "matrix": [2, 10], "x": 8.5, "y": 3.25},
                {"label": "O", "matrix": [2, 11], "x": 9.5, "y": 3.25},
                {"label": "P", "matrix": [2, 12], "x": 10.5, "y": 3.25},
                {"label": "{", "matrix": [2, 13], "x": 11.5, "y": 3.25},
                {"label": "}", "matrix": [2, 14], "x": 12.5, "y": 3.25},
                {"label": "|", "matrix": [2, 15], "x": 13.5, "y": 3.25, "w": 1.5},
                {"label": "Delete", "matrix": [3, 0], "x": 15, "y": 3.25},

                {"label": "Caps Lock", "matrix": [3, 1], "x": 0, "y": 4.25, "w": 1.75},
                {"label": "A", "matrix": [3, 2], "x": 1.75, "y": 4.25},
                {"label": "S", "matrix": [3, 3], "x": 2.75, "y": 4.25},
                {"label": "D", "matrix": [3, 4], "x": 3.75, "y": 4.25},
                {"label": "F", "matrix": [3, 5], "x": 4.75, "y": 4.25},
                {"label": "G", "matrix": [3, 6], "x": 5.75, "y": 4.25},
                {"label": "H", "matrix": [3, 7], "x": 6.75, "y": 4.25},
                {"label": "J", "matrix": [3, 8], "x": 7.75, "y": 4.25},
                {"label": "K", "matrix": [3, 9], "x": 8.75, "y": 4.25},
                {"label": "L", "matrix": [3, 10], "x": 9.75, "y": 4.25},
                {"label": ";", "matrix": [3, 11], "x": 10.75, "y": 4.25},
                {"label": "\"", "matrix": [3, 12], "x": 11.75, "y": 4.25},
                {"label": "NUHS", "matrix": [3, 13], "x": 12.75, "y": 4.25},
                {"label": "Enter", "matrix": [3, 14], "x": 13.75, "y": 3.25, "w": 1.25, "h": 2},
                {"label": "PgUp", "matrix": [3, 15], "x": 15, "y": 4.25},

                {"label": "Shift", "matrix": [4, 0], "x": 0, "y": 5.25, "w": 1.25},
                {"label": "|", "matrix": [4, 1], "x": 1.25, "y": 5.25},
                {"label": "Z", "matrix": [4, 2], "x": 2.25, "y": 5.25},
                {"label": "X", "matrix": [4, 3], "x": 3.25, "y": 5.25},
                {"label": "C", "matrix": [4, 4], "x": 4.25, "y": 5.25},
                {"label": "V", "matrix": [4, 5], "x": 5.25, "y": 5.25},
                {"label": "B", "matrix": [4, 6], "x": 6.25, "y": 5.25},
                {"label": "N", "matrix": [4, 7], "x": 7.25, "y": 5.25},
                {"label": "M", "matrix": [4, 8], "x": 8.25, "y": 5.25},
                {"label": "<", "matrix": [4, 9], "x": 9.25, "y": 5.25},
                {"label": ">", "matrix": [4, 10], "x": 10.25, "y": 5.25},
                {"label": "?", "matrix": [4, 11], "x": 11.25, "y": 5.25},
                {"label": "Shift", "matrix": [4, 12], "x": 12.25, "y": 5.25, "w": 1.75},
                {"label": "Up", "matrix": [4, 13], "x": 14, "y": 5.25},
                {"label": "PgDn", "matrix": [4, 14], "x": 15, "y": 5.25},

                {"label": "Ctrl", "matrix": [5, 0], "x": 0, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 1], "x": 1.25, "y": 6.25, "w": 1.25},
                {"label": "Cmd", "matrix": [5, 2], "x": 2.5, "y": 6.25, "w": 1.25},
                {"label": "Space", "matrix": [5, 6], "x": 3.75, "y": 6.25, "w": 6.25},
                {"label": "Cmd", "matrix": [5, 10], "x": 10, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 11], "x": 11.25, "y": 6.25, "w": 1.25},

                {"label": "Left", "matrix": [5, 12], "x": 13, "y": 6.25},
                {"label": "Down", "matrix": [5, 13], "x": 14, "y": 6.25},
                {"label": "Right", "matrix": [5, 14], "x": 15, "y": 6.25}
            ]
        },
        "LAYOUT_iso_blocker_split_space": {
            "layout": [
                {"label": "ENC1", "matrix": [2, 0], "x": 13.5, "y": 0},
                {"label": "ENC2", "matrix": [2, 1], "x": 15, "y": 0},

                {"label": "Esc", "matrix": [0, 0], "x": 0, "y": 1},
                {"label": "F1", "matrix": [0, 1], "x": 1.5, "y": 1},
                {"label": "F2", "matrix": [0, 2], "x": 2.5, "y": 1},
                {"label": "F3", "matrix": [0, 3], "x": 3.5, "y": 1},
                {"label": "F4", "matrix": [0, 4], "x": 4.5, "y": 1},
                {"label": "F5", "matrix": [0, 5], "x": 6, "y": 1},
                {"label": "F6", "matrix": [0, 6], "x": 7, "y": 1},
                {"label": "F7", "matrix": [0, 7], "x": 8, "y": 1},
                {"label": "F8", "matrix": [0, 8], "x": 9, "y": 1},
                {"label": "F9", "matrix": [0, 9], "x": 10.5, "y": 1},
                {"label": "F10", "matrix": [0, 10], "x": 11.5, "y": 1},
                {"label": "F11", "matrix": [0, 11], "x": 12.5, "y": 1},
                {"label": "F12", "matrix": [0, 12], "x": 13.5, "y": 1},
                {"label": "F13", "matrix": [0, 13], "x": 15, "y": 1},

                {"label": "`", "matrix": [1, 0], "x": 0, "y": 2.25},
                {"label": "1", "matrix": [1, 1], "x": 1, "y": 2.25},
                {"label": "2", "matrix": [1, 2], "x": 2, "y": 2.25},
                {"label": "3", "matrix": [1, 3], "x": 3, "y": 2.25},
                {"label": "4", "matrix": [1, 4], "x": 4, "y": 2.25},
                {"label": "5", "matrix": [1, 5], "x": 5, "y": 2.25},
                {"label": "6", "matrix": [1, 6], "x": 6, "y": 2.25},
                {"label": "7", "matrix": [1, 7], "x": 7, "y": 2.25},
                {"label": "8", "matrix": [1, 8], "x": 8, "y": 2.25},
                {"label": "9", "matrix": [1, 9], "x": 9, "y": 2.25},
                {"label": "0", "matrix": [1, 10], "x": 10, "y": 2.25},
                {"label": "-", "matrix": [1, 11], "x": 11, "y": 2.25},
                {"label": "=", "matrix": [1, 12], "x": 12, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 14], "x": 13, "y": 2.25, "w": 2},
                {"label": "Home", "matrix": [1, 15], "x": 15, "y": 2.25},      

                {"label": "Tab", "matrix": [2, 2], "x": 0, "y": 3.25, "w": 1.5},
                {"label": "Q", "matrix": [2, 3], "x": 1.5, "y": 3.25},
                {"label": "W", "matrix": [2, 4], "x": 2.5, "y": 3.25},
                {"label": "E", "matrix": [2, 5], "x": 3.5, "y": 3.25},
                {"label": "R", "matrix": [2, 6], "x": 4.5, "y": 3.25},
                {"label": "T", "matrix": [2, 7], "x": 5.5, "y": 3.25},
                {"label": "Y", "matrix": [2, 8], "x": 6.5, "y": 3.25},
                {"label": "U", "matrix": [2, 9], "x": 7.5, "y": 3.25},
                {"label": "I", "matrix": [2, 10], "x": 8.5, "y": 3.25},
                {"label": "O", "matrix": [2, 11], "x": 9.5, "y": 3.25},
                {"label": "P", "matrix": [2, 12], "x": 10.5, "y": 3.25},
                {"label": "{", "matrix": [2, 13], "x": 11.5, "y": 3.25},
                {"label": "}", "matrix": [2, 14], "x": 12.5, "y": 3.25},
                {"label": "|", "matrix": [2, 15], "x": 13.5, "y": 3.25, "w": 1.5},
                {"label": "Delete", "matrix": [3, 0], "x": 15, "y": 3.25},

                {"label": "Caps Lock", "matrix": [3, 1], "x": 0, "y": 4.25, "w": 1.75},
                {"label": "A", "matrix": [3, 2], "x": 1.75, "y": 4.25},
                {"label": "S", "matrix": [3, 3], "x": 2.75, "y": 4.25},
                {"label": "D", "matrix": [3, 4], "x": 3.75, "y": 4.25},
                {"label": "F", "matrix": [3, 5], "x": 4.75, "y": 4.25},
                {"label": "G", "matrix": [3, 6], "x": 5.75, "y": 4.25},
                {"label": "H", "matrix": [3, 7], "x": 6.75, "y": 4.25},
                {"label": "J", "matrix": [3, 8], "x": 7.75, "y": 4.25},
                {"label": "K", "matrix": [3, 9], "x": 8.75, "y": 4.25},
                {"label": "L", "matrix": [3, 10], "x": 9.75, "y": 4.25},
                {"label": ";", "matrix": [3, 11], "x": 10.75, "y": 4.25},
                {"label": "\"", "matrix": [3, 12], "x": 11.75, "y": 4.25},
                {"label": "NUHS", "matrix": [3, 13], "x": 12.75, "y": 4.25},
                {"label": "Enter", "matrix": [3, 14], "x": 13.75, "y": 3.25, "w": 1.25, "h": 2},
                {"label": "PgUp", "matrix": [3, 15], "x": 15, "y": 4.25},

                {"label": "Shift", "matrix": [4, 0], "x": 0, "y": 5.25, "w": 1.25},
                {"label": "|", "matrix": [4, 1], "x": 1.25, "y": 5.25},
                {"label": "Z", "matrix": [4, 2], "x": 2.25, "y": 5.25},
                {"label": "X", "matrix": [4, 3], "x": 3.25, "y": 5.25},
                {"label": "C", "matrix": [4, 4], "x": 4.25, "y": 5.25},
                {"label": "V", "matrix": [4, 5], "x": 5.25, "y": 5.25},
                {"label": "B", "matrix": [4, 6], "x": 6.25, "y": 5.25},
                {"label": "N", "matrix": [4, 7], "x": 7.25, "y": 5.25},
                {"label": "M", "matrix": [4, 8], "x": 8.25, "y": 5.25},
                {"label": "<", "matrix": [4, 9], "x": 9.25, "y": 5.25},
                {"label": ">", "matrix": [4, 10], "x": 10.25, "y": 5.25},
                {"label": "?", "matrix": [4, 11], "x": 11.25, "y": 5.25},
                {"label": "Shift", "matrix": [4, 12], "x": 12.25, "y": 5.25, "w": 1.75},
                {"label": "Up", "matrix": [4, 13], "x": 14, "y": 5.25},
                {"label": "PgDn", "matrix": [4, 14], "x": 15, "y": 5.25},

                {"label": "Ctrl", "matrix": [5, 0], "x": 0, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 1], "x": 1.25, "y": 6.25, "w": 1.25},
                {"label": "Cmd", "matrix": [5, 2], "x": 2.5, "y": 6.25, "w": 1.25},
                {"label": "Space", "matrix": [5, 4], "x": 3.75, "y": 6.25, "w": 2.25},
                {"label": "Super", "matrix": [5, 6], "x": 6, "y": 6.25, "w": 1.25},
                {"label": "Space", "matrix": [5, 8], "x": 7.25, "y": 6.25, "w": 2.75},
                {"label": "Cmd", "matrix": [5, 10], "x": 10, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 11], "x": 11.25, "y": 6.25, "w": 1.25},

                {"label": "Left", "matrix": [5, 12], "x": 13, "y": 6.25},
                {"label": "Down", "matrix": [5, 13], "x": 14, "y": 6.25},
                {"label": "Right", "matrix": [5, 14], "x": 15, "y": 6.25}
            ]
        },
        "LAYOUT_iso_blocker_split_bs_space": {
            "layout": [
                {"label": "ENC1", "matrix": [2, 0], "x": 13.5, "y": 0},
                {"label": "ENC2", "matrix": [2, 1], "x": 15, "y": 0},

                {"label": "Esc", "matrix": [0, 0], "x": 0, "y": 1},
                {"label": "F1", "matrix": [0, 1], "x": 1.5, "y": 1},
                {"label": "F2", "matrix": [0, 2], "x": 2.5, "y": 1},
                {"label": "F3", "matrix": [0, 3], "x": 3.5, "y": 1},
                {"label": "F4", "matrix": [0, 4], "x": 4.5, "y": 1},
                {"label": "F5", "matrix": [0, 5], "x": 6, "y": 1},
                {"label": "F6", "matrix": [0, 6], "x": 7, "y": 1},
                {"label": "F7", "matrix": [0, 7], "x": 8, "y": 1},
                {"label": "F8", "matrix": [0, 8], "x": 9, "y": 1},
                {"label": "F9", "matrix": [0, 9], "x": 10.5, "y": 1},
                {"label": "F10", "matrix": [0, 10], "x": 11.5, "y": 1},
                {"label": "F11", "matrix": [0, 11], "x": 12.5, "y": 1},
                {"label": "F12", "matrix": [0, 12], "x": 13.5, "y": 1},
                {"label": "F13", "matrix": [0, 13], "x": 15, "y": 1},

                {"label": "`", "matrix": [1, 0], "x": 0, "y": 2.25},
                {"label": "1", "matrix": [1, 1], "x": 1, "y": 2.25},
                {"label": "2", "matrix": [1, 2], "x": 2, "y": 2.25},
                {"label": "3", "matrix": [1, 3], "x": 3, "y": 2.25},
                {"label": "4", "matrix": [1, 4], "x": 4, "y": 2.25},
                {"label": "5", "matrix": [1, 5], "x": 5, "y": 2.25},
                {"label": "6", "matrix": [1, 6], "x": 6, "y": 2.25},
                {"label": "7", "matrix": [1, 7], "x": 7, "y": 2.25},
                {"label": "8", "matrix": [1, 8], "x": 8, "y": 2.25},
                {"label": "9", "matrix": [1, 9], "x": 9, "y": 2.25},
                {"label": "0", "matrix": [1, 10], "x": 10, "y": 2.25},
                {"label": "-", "matrix": [1, 11], "x": 11, "y": 2.25},
                {"label": "=", "matrix": [1, 12], "x": 12, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 13], "x": 13, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 14], "x": 14, "y": 2.25},
                {"label": "Home", "matrix": [1, 15], "x": 15, "y": 2.25},      

                {"label": "Tab", "matrix": [2, 2], "x": 0, "y": 3.25, "w": 1.5},
                {"label": "Q", "matrix": [2, 3], "x": 1.5, "y": 3.25},
                {"label": "W", "matrix": [2, 4], "x": 2.5, "y": 3.25},
                {"label": "E", "matrix": [2, 5], "x": 3.5, "y": 3.25},
                {"label": "R", "matrix": [2, 6], "x": 4.5, "y": 3.25},
                {"label": "T", "matrix": [2, 7], "x": 5.5, "y": 3.25},
                {"label": "Y", "matrix": [2, 8], "x": 6.5, "y": 3.25},
                {"label": "U", "matrix": [2, 9], "x": 7.5, "y": 3.25},
                {"label": "I", "matrix": [2, 10], "x": 8.5, "y": 3.25},
                {"label": "O", "matrix": [2, 11], "x": 9.5, "y": 3.25},
                {"label": "P", "matrix": [2, 12], "x": 10.5, "y": 3.25},
                {"label": "{", "matrix": [2, 13], "x": 11.5, "y": 3.25},
                {"label": "}", "matrix": [2, 14], "x": 12.5, "y": 3.25},
                {"label": "|", "matrix": [2, 15], "x": 13.5, "y": 3.25, "w": 1.5},
                {"label": "Delete", "matrix": [3, 0], "x": 15, "y": 3.25},

                {"label": "Caps Lock", "matrix": [3, 1], "x": 0, "y": 4.25, "w": 1.75},
                {"label": "A", "matrix": [3, 2], "x": 1.75, "y": 4.25},
                {"label": "S", "matrix": [3, 3], "x": 2.75, "y": 4.25},
                {"label": "D", "matrix": [3, 4], "x": 3.75, "y": 4.25},
                {"label": "F", "matrix": [3, 5], "x": 4.75, "y": 4.25},
                {"label": "G", "matrix": [3, 6], "x": 5.75, "y": 4.25},
                {"label": "H", "matrix": [3, 7], "x": 6.75, "y": 4.25},
                {"label": "J", "matrix": [3, 8], "x": 7.75, "y": 4.25},
                {"label": "K", "matrix": [3, 9], "x": 8.75, "y": 4.25},
                {"label": "L", "matrix": [3, 10], "x": 9.75, "y": 4.25},
                {"label": ";", "matrix": [3, 11], "x": 10.75, "y": 4.25},
                {"label": "\"", "matrix": [3, 12], "x": 11.75, "y": 4.25},
                {"label": "NUHS", "matrix": [3, 13], "x": 12.75, "y": 4.25},
                {"label": "Enter", "matrix": [3, 14], "x": 13.75, "y": 3.25, "w": 1.25, "h": 2},
                {"label": "PgUp", "matrix": [3, 15], "x": 15, "y": 4.25},

                {"label": "Shift", "matrix": [4, 0], "x": 0, "y": 5.25, "w": 1.25},
                {"label": "|", "matrix": [4, 1], "x": 1.25, "y": 5.25},
                {"label": "Z", "matrix": [4, 2], "x": 2.25, "y": 5.25},
                {"label": "X", "matrix": [4, 3], "x": 3.25, "y": 5.25},
                {"label": "C", "matrix": [4, 4], "x": 4.25, "y": 5.25},
                {"label": "V", "matrix": [4, 5], "x": 5.25, "y": 5.25},
                {"label": "B", "matrix": [4, 6], "x": 6.25, "y": 5.25},
                {"label": "N", "matrix": [4, 7], "x": 7.25, "y": 5.25},
                {"label": "M", "matrix": [4, 8], "x": 8.25, "y": 5.25},
                {"label": "<", "matrix": [4, 9], "x": 9.25, "y": 5.25},
                {"label": ">", "matrix": [4, 10], "x": 10.25, "y": 5.25},
                {"label": "?", "matrix": [4, 11], "x": 11.25, "y": 5.25},
                {"label": "Shift", "matrix": [4, 12], "x": 12.25, "y": 5.25, "w": 1.75},
                {"label": "Up", "matrix": [4, 13], "x": 14, "y": 5.25},
                {"label": "PgDn", "matrix": [4, 14], "x": 15, "y": 5.25},

                {"label": "Ctrl", "matrix": [5, 0], "x": 0, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 1], "x": 1.25, "y": 6.25, "w": 1.25},
                {"label": "Cmd", "matrix": [5, 2], "x": 2.5, "y": 6.25, "w": 1.25},
                {"label": "Space", "matrix": [5, 4], "x": 3.75, "y": 6.25, "w": 2.25},
                {"label": "Super", "matrix": [5, 6], "x": 6, "y": 6.25, "w": 1.25},
                {"label": "Space", "matrix": [5, 8], "x": 7.25, "y": 6.25, "w": 2.75},
                {"label": "Cmd", "matrix": [5, 10], "x": 10, "y": 6.25, "w": 1.25},
                {"label": "Alt", "matrix": [5, 11], "x": 11.25, "y": 6.25, "w": 1.25},

                {"label": "Left", "matrix": [5, 12], "x": 13, "y": 6.25},
                {"label": "Down", "matrix": [5, 13], "x": 14, "y": 6.25},
                {"label": "Right", "matrix": [5, 14], "x": 15, "y": 6.25}
            ]
        },
        "LAYOUT_iso_blocker_tsangan": {
            "layout": [
                {"label": "ENC1", "matrix": [2, 0], "x": 13.5, "y": 0},
                {"label": "ENC2", "matrix": [2, 1], "x": 15, "y": 0},

                {"label": "Esc", "matrix": [0, 0], "x": 0, "y": 1},
                {"label": "F1", "matrix": [0, 1], "x": 1.5, "y": 1},
                {"label": "F2", "matrix": [0, 2], "x": 2.5, "y": 1},
                {"label": "F3", "matrix": [0, 3], "x": 3.5, "y": 1},
                {"label": "F4", "matrix": [0, 4], "x": 4.5, "y": 1},
                {"label": "F5", "matrix": [0, 5], "x": 6, "y": 1},
                {"label": "F6", "matrix": [0, 6], "x": 7, "y": 1},
                {"label": "F7", "matrix": [0, 7], "x": 8, "y": 1},
                {"label": "F8", "matrix": [0, 8], "x": 9, "y": 1},
                {"label": "F9", "matrix": [0, 9], "x": 10.5, "y": 1},
                {"label": "F10", "matrix": [0, 10], "x": 11.5, "y": 1},
                {"label": "F11", "matrix": [0, 11], "x": 12.5, "y": 1},
                {"label": "F12", "matrix": [0, 12], "x": 13.5, "y": 1},
                {"label": "F13", "matrix": [0, 13], "x": 15, "y": 1},

                {"label": "`", "matrix": [1, 0], "x": 0, "y": 2.25},
                {"label": "1", "matrix": [1, 1], "x": 1, "y": 2.25},
                {"label": "2", "matrix": [1, 2], "x": 2, "y": 2.25},
                {"label": "3", "matrix": [1, 3], "x": 3, "y": 2.25},
                {"label": "4", "matrix": [1, 4], "x": 4, "y": 2.25},
                {"label": "5", "matrix": [1, 5], "x": 5, "y": 2.25},
                {"label": "6", "matrix": [1, 6], "x": 6, "y": 2.25},
                {"label": "7", "matrix": [1, 7], "x": 7, "y": 2.25},
                {"label": "8", "matrix": [1, 8], "x": 8, "y": 2.25},
                {"label": "9", "matrix": [1, 9], "x": 9, "y": 2.25},
                {"label": "0", "matrix": [1, 10], "x": 10, "y": 2.25},
                {"label": "-", "matrix": [1, 11], "x": 11, "y": 2.25},
                {"label": "=", "matrix": [1, 12], "x": 12, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 14], "x": 13, "y": 2.25, "w": 2},
                {"label": "Home", "matrix": [1, 15], "x": 15, "y": 2.25},      

                {"label": "Tab", "matrix": [2, 2], "x": 0, "y": 3.25, "w": 1.5},
                {"label": "Q", "matrix": [2, 3], "x": 1.5, "y": 3.25},
                {"label": "W", "matrix": [2, 4], "x": 2.5, "y": 3.25},
                {"label": "E", "matrix": [2, 5], "x": 3.5, "y": 3.25},
                {"label": "R", "matrix": [2, 6], "x": 4.5, "y": 3.25},
                {"label": "T", "matrix": [2, 7], "x": 5.5, "y": 3.25},
                {"label": "Y", "matrix": [2, 8], "x": 6.5, "y": 3.25},
                {"label": "U", "matrix": [2, 9], "x": 7.5, "y": 3.25},
                {"label": "I", "matrix": [2, 10], "x": 8.5, "y": 3.25},
                {"label": "O", "matrix": [2, 11], "x": 9.5, "y": 3.25},
                {"label": "P", "matrix": [2, 12], "x": 10.5, "y": 3.25},
                {"label": "{", "matrix": [2, 13], "x": 11.5, "y": 3.25},
                {"label": "}", "matrix": [2, 14], "x": 12.5, "y": 3.25},
                {"label": "|", "matrix": [2, 15], "x": 13.5, "y": 3.25, "w": 1.5},
                {"label": "Delete", "matrix": [3, 0], "x": 15, "y": 3.25},

                {"label": "Caps Lock", "matrix": [3, 1], "x": 0, "y": 4.25, "w": 1.75},
                {"label": "A", "matrix": [3, 2], "x": 1.75, "y": 4.25},
                {"label": "S", "matrix": [3, 3], "x": 2.75, "y": 4.25},
                {"label": "D", "matrix": [3, 4], "x": 3.75, "y": 4.25},
                {"label": "F", "matrix": [3, 5], "x": 4.75, "y": 4.25},
                {"label": "G", "matrix": [3, 6], "x": 5.75, "y": 4.25},
                {"label": "H", "matrix": [3, 7], "x": 6.75, "y": 4.25},
                {"label": "J", "matrix": [3, 8], "x": 7.75, "y": 4.25},
                {"label": "K", "matrix": [3, 9], "x": 8.75, "y": 4.25},
                {"label": "L", "matrix": [3, 10], "x": 9.75, "y": 4.25},
                {"label": ";", "matrix": [3, 11], "x": 10.75, "y": 4.25},
                {"label": "\"", "matrix": [3, 12], "x": 11.75, "y": 4.25},
                {"label": "NUHS", "matrix": [3, 13], "x": 12.75, "y": 4.25},
                {"label": "Enter", "matrix": [3, 14], "x": 13.75, "y": 3.25, "w": 1.25, "h": 2},
                {"label": "PgUp", "matrix": [3, 15], "x": 15, "y": 4.25},

                {"label": "Shift", "matrix": [4, 0], "x": 0, "y": 5.25, "w": 1.25},
                {"label": "|", "matrix": [4, 1], "x": 1.25, "y": 5.25},
                {"label": "Z", "matrix": [4, 2], "x": 2.25, "y": 5.25},
                {"label": "X", "matrix": [4, 3], "x": 3.25, "y": 5.25},
                {"label": "C", "matrix": [4, 4], "x": 4.25, "y": 5.25},
                {"label": "V", "matrix": [4, 5], "x": 5.25, "y": 5.25},
                {"label": "B", "matrix": [4, 6], "x": 6.25, "y": 5.25},
                {"label": "N", "matrix": [4, 7], "x": 7.25, "y": 5.25},
                {"label": "M", "matrix": [4, 8], "x": 8.25, "y": 5.25},
                {"label": "<", "matrix": [4, 9], "x": 9.25, "y": 5.25},
                {"label": ">", "matrix": [4, 10], "x": 10.25, "y": 5.25},
                {"label": "?", "matrix": [4, 11], "x": 11.25, "y": 5.25},
                {"label": "Shift", "matrix": [4, 12], "x": 12.25, "y": 5.25, "w": 1.75},
                {"label": "Up", "matrix": [4, 13], "x": 14, "y": 5.25},
                {"label": "PgDn", "matrix": [4, 14], "x": 15, "y": 5.25},

                {"label": "Ctrl", "matrix": [5, 0], "x": 0, "y": 6.25, "w": 1.5},
                {"label": "Alt", "matrix": [5, 1], "x": 1.5, "y": 6.25},
                {"label": "Cmd", "matrix": [5, 2], "x": 2.5, "y": 6.25, "w": 1.5},
                {"label": "Space", "matrix": [5, 6], "x": 4, "y": 6.25, "w": 7},
                {"label": "Cmd", "matrix": [5, 11], "x": 11, "y": 6.25, "w": 1.5},

                {"label": "Left", "matrix": [5, 12], "x": 13, "y": 6.25},
                {"label": "Down", "matrix": [5, 13], "x": 14, "y": 6.25},
                {"label": "Right", "matrix": [5, 14], "x": 15, "y": 6.25}
            ]
        },
        "LAYOUT_iso_blocker_tsangan_split_bs": {
            "layout": [
                {"label": "ENC1", "matrix": [2, 0], "x": 13.5, "y": 0},
                {"label": "ENC2", "matrix": [2, 1], "x": 15, "y": 0},

                {"label": "Esc", "matrix": [0, 0], "x": 0, "y": 1},
                {"label": "F1", "matrix": [0, 1], "x": 1.5, "y": 1},
                {"label": "F2", "matrix": [0, 2], "x": 2.5, "y": 1},
                {"label": "F3", "matrix": [0, 3], "x": 3.5, "y": 1},
                {"label": "F4", "matrix": [0, 4], "x": 4.5, "y": 1},
                {"label": "F5", "matrix": [0, 5], "x": 6, "y": 1},
                {"label": "F6", "matrix": [0, 6], "x": 7, "y": 1},
                {"label": "F7", "matrix": [0, 7], "x": 8, "y": 1},
                {"label": "F8", "matrix": [0, 8], "x": 9, "y": 1},
                {"label": "F9", "matrix": [0, 9], "x": 10.5, "y": 1},
                {"label": "F10", "matrix": [0, 10], "x": 11.5, "y": 1},
                {"label": "F11", "matrix": [0, 11], "x": 12.5, "y": 1},
                {"label": "F12", "matrix": [0, 12], "x": 13.5, "y": 1},
                {"label": "F13", "matrix": [0, 13], "x": 15, "y": 1},

                {"label": "`", "matrix": [1, 0], "x": 0, "y": 2.25},
                {"label": "1", "matrix": [1, 1], "x": 1, "y": 2.25},
                {"label": "2", "matrix": [1, 2], "x": 2, "y": 2.25},
                {"label": "3", "matrix": [1, 3], "x": 3, "y": 2.25},
                {"label": "4", "matrix": [1, 4], "x": 4, "y": 2.25},
                {"label": "5", "matrix": [1, 5], "x": 5, "y": 2.25},
                {"label": "6", "matrix": [1, 6], "x": 6, "y": 2.25},
                {"label": "7", "matrix": [1, 7], "x": 7, "y": 2.25},
                {"label": "8", "matrix": [1, 8], "x": 8, "y": 2.25},
                {"label": "9", "matrix": [1, 9], "x": 9, "y": 2.25},
                {"label": "0", "matrix": [1, 10], "x": 10, "y": 2.25},
                {"label": "-", "matrix": [1, 11], "x": 11, "y": 2.25},
                {"label": "=", "matrix": [1, 12], "x": 12, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 13], "x": 13, "y": 2.25},
                {"label": "Backspace", "matrix": [1, 14], "x": 14, "y": 2.25},
                {"label": "Home", "matrix": [1, 15], "x": 15, "y": 2.25},      

                {"label": "Tab", "matrix": [2, 2], "x": 0, "y": 3.25, "w": 1.5},
                {"label": "Q", "matrix": [2, 3], "x": 1.5, "y": 3.25},
                {"label": "W", "matrix": [2, 4], "x": 2.5, "y": 3.25},
                {"label": "E", "matrix": [2, 5], "x": 3.5, "y": 3.25},
                {"label": "R", "matrix": [2, 6], "x": 4.5, "y": 3.25},
                {"label": "T", "matrix": [2, 7], "x": 5.5, "y": 3.25},
                {"label": "Y", "matrix": [2, 8], "x": 6.5, "y": 3.25},
                {"label": "U", "matrix": [2, 9], "x": 7.5, "y": 3.25},
                {"label": "I", "matrix": [2, 10], "x": 8.5, "y": 3.25},
                {"label": "O", "matrix": [2, 11], "x": 9.5, "y": 3.25},
                {"label": "P", "matrix": [2, 12], "x": 10.5, "y": 3.25},
                {"label": "{", "matrix": [2, 13], "x": 11.5, "y": 3.25},
                {"label": "}", "matrix": [2, 14], "x": 12.5, "y": 3.25},
                {"label": "|", "matrix": [2, 15], "x": 13.5, "y": 3.25, "w": 1.5},
                {"label": "Delete", "matrix": [3, 0], "x": 15, "y": 3.25},

                {"label": "Caps Lock", "matrix": [3, 1], "x": 0, "y": 4.25, "w": 1.75},
                {"label": "A", "matrix": [3, 2], "x": 1.75, "y": 4.25},
                {"label": "S", "matrix": [3, 3], "x": 2.75, "y": 4.25},
                {"label": "D", "matrix": [3, 4], "x": 3.75, "y": 4.25},
                {"label": "F", "matrix": [3, 5], "x": 4.75, "y": 4.25},
                {"label": "G", "matrix": [3, 6], "x": 5.75, "y": 4.25},
                {"label": "H", "matrix": [3, 7], "x": 6.75, "y": 4.25},
                {"label": "J", "matrix": [3, 8], "x": 7.75, "y": 4.25},
                {"label": "K", "matrix": [3, 9], "x": 8.75, "y": 4.25},
                {"label": "L", "matrix": [3, 10], "x": 9.75, "y": 4.25},
                {"label": ";", "matrix": [3, 11], "x": 10.75, "y": 4.25},
                {"label": "\"", "matrix": [3, 12], "x": 11.75, "y": 4.25},
                {"label": "NUHS", "matrix": [3, 13], "x": 12.75, "y": 4.25},
                {"label": "Enter", "matrix": [3, 14], "x": 13.75, "y": 3.25, "w": 1.25, "h": 2},
                {"label": "PgUp", "matrix": [3, 15], "x": 15, "y": 4.25},

                {"label": "Shift", "matrix": [4, 0], "x": 0, "y": 5.25, "w": 1.25},
                {"label": "|", "matrix": [4, 1], "x": 1.25, "y": 5.25},
                {"label": "Z", "matrix": [4, 2], "x": 2.25, "y": 5.25},
                {"label": "X", "matrix": [4, 3], "x": 3.25, "y": 5.25},
                {"label": "C", "matrix": [4, 4], "x": 4.25, "y": 5.25},
                {"label": "V", "matrix": [4, 5], "x": 5.25, "y": 5.25},
                {"label": "B", "matrix": [4, 6], "x": 6.25, "y": 5.25},
                {"label": "N", "matrix": [4, 7], "x": 7.25, "y": 5.25},
                {"label": "M", "matrix": [4, 8], "x": 8.25, "y": 5.25},
                {"label": "<", "matrix": [4, 9], "x": 9.25, "y": 5.25},
                {"label": ">", "matrix": [4, 10], "x": 10.25, "y": 5.25},
                {"label": "?", "matrix": [4, 11], "x": 11.25, "y": 5.25},
                {"label": "Shift", "matrix": [4, 12], "x": 12.25, "y": 5.25, "w": 1.75},
                {"label": "Up", "matrix": [4, 13], "x": 14, "y": 5.25},
                {"label": "PgDn", "matrix": [4, 14], "x": 15, "y": 5.25},

                {"label": "Ctrl", "matrix": [5, 0], "x": 0, "y": 6.25, "w": 1.5},
                {"label": "Alt", "matrix": [5, 1], "x": 1.5, "y": 6.25},
                {"label": "Cmd", "matrix": [5, 2], "x": 2.5, "y": 6.25, "w": 1.5},
                {"label": "Space", "matrix": [5, 6], "x": 4, "y": 6.25, "w": 7},
                {"label": "Cmd", "matrix": [5, 11], "x": 11, "y": 6.25, "w": 1.5},

                {"label": "Left", "matrix": [5, 12], "x": 13, "y": 6.25},
                {"label": "Down", "matrix": [5, 13], "x": 14, "y": 6.25},
                {"label": "Right", "matrix": [5, 14], "x": 15, "y": 6.25}
            ]
        }
    }
}
```